docs/local-development.md

```diff
@@ -73,15 +73,13 @@ To recompile, kill the process with ctrl-C and re-run it.
 
 ### Define and create OpenStack credentials
 
-Create a `clouds.yaml` file in `orc/examples/credentials`. The name of the cloud in this clouds.yaml must be `openstack`.
+Create a `clouds.yaml` file in `examples/bases/credentials`. The name of the cloud in this clouds.yaml must be `openstack`.
 
 This file is in both `.gitignore` and `.dockerignore`, so should not be accidentally added to the git repo or a container build.
 
-Create a credentials secret in your development cluster by loading the
-`credentials-only` kustomize resource:
-
+Create a credentials secret in your development cluster by running the following from the examples directory:
 ```bash
-$ kubectl apply -k orc/examples/credentials-only --server-side
+$ make load-credentials
```

Comment:
Have you checked that with your changes the order is still correct? We recently broke it and I had to swap the creation of dev-config with the loading of credentials. See https://github.com/k-orc/openstack-resource-controller/pull/146.

Reply:
Yeah, I checked it.

Reply:
Oh, wait. I see what you mean. No, that's not going to work. They still need to be swapped. I think it's easier for me to update this one because it's contains function changes. Lets drop https://github.com/k-orc/openstack-resource-controller/pull/146.